name.abuchen.portfolio.tests/src/scenarios/VolatilityTestCase.java

```diff
@@ -35,48 +35,52 @@ public static void loadClientFile() throws IOException
     @Test
     public void testVolatilityOfSharesHeldIsIdenticalToExcel() throws IOException
     {
-        ReportingPeriod report = new ReportingPeriod.FromXtoY(LocalDate.parse("2014-01-31"), LocalDate.parse("2014-07-31"));
+        ReportingPeriod report = new ReportingPeriod.FromXtoY(LocalDate.parse("2014-01-31"),
+                        LocalDate.parse("2014-07-31"));
         List<Exception> warnings = new ArrayList<>();
         PerformanceIndex index = PerformanceIndex.forClient(client, converter, report, warnings);
 
         assertThat(warnings, empty());
-        assertThat(index.getVolatility().getStandardDeviation(), closeTo(0.141568791460, 0.1e-10)); // excel
```

Comment:
Diese Werte kamen ursprünglich aus der Excel Datei die ebenfalls in dem Verzeichnis eingecheckt ist.
Wenn wir die hier ändern, sollte auch die "Referenzrechnung" geändert werden.
Lass uns aber abwarten was wir mit den Handelstagen in Deutschland machen - ich gehe davon aus, dass die Volatilität niedriger ist weil jetzt ein handelsfreier Tag in die Rechnung einfliesst.

Reply:
In die Berechnung v. Excel muss ich wohl noch etwas tiefer einsteigen, den Excel kommt auf ein minimal anderes Ergebnis (nach Bereinigung d. Feiertage).